samples/subsys/power/stm32_pm/lowpower/Kconfig

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2019 STMicroelectronics
```

Comment:
I wonder if a prj.conf would be enough here

Reply:
yes, it works without KConfig